packages/bpk-docs/src/pages/BannerAlertsPage/BannerAlertsPage.js

```diff
@@ -32,6 +35,62 @@ sapien, et dapibus mi aliquet non. Pellentesque auctor sagittis lectus vitae rho
 ante in, vestibulum nulla.`;
 const richMessage = <span style={{ fontWeight: 700 }}>Successful alert with custom rendered message</span>;
```

Comment:
same here, can you please use the token `FONT_WEIGHT_BOLD` for the `fontWeight`?